src/Qir/Runtime/lib/QIR/callables.cpp

```diff
@@ -370,7 +370,8 @@ QirTupleHeader* FlattenControlArrays(QirTupleHeader* tuple, int depth)
 
         QirArray* controls = current->controls;
         const size_t blockSize = qubitSize * controls->count;
-        assert(dst + blockSize <= dstEnd);
+        assert(dst + blockSize <= dstEnd); 
+        (void)dstEnd;   // Calm down the "Unused Var" warning in Release build (`assert()` is Debug only).

```

Comment:
Food for thought: A pattern I've seen in other codebases is to create a macro named `UNUSED` or `UNREFERENCED_ARG` or something similar that does this, just to make the intent clear. Your comment here is clear enough such that I don't think this is really a problem and there aren't that many places we need this pattern (and I'm not certain they all share a convenient header to define that macro in), so feel free to ignore.

Reply:
Good point, thank you! 
I'm adding `UNUSED()`.